nbs/tsdataset.ipynb

```diff
@@ -54,11 +54,12 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #97.                total_rows = rg.num_rows
</pre><p>If we enforce the convention of saving the series sorted by time we could also simplify this a bit by getting the maximum of the last row group:</p>

<pre class="ql-syntax" spellcheck="false">last_time = meta.row_group(meta.num_row_groups -1).column(col2pos[time_col]).statistics.max
</pre><p>Also we could get the total rows with this:</p>

<pre class="ql-syntax" spellcheck="false">total_rows = sum(meta.row_group(i).num_rows for i in range(meta.num_row_groups))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>